service/controller/clusterapi/v28/cloudconfig/cloud_config.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 const (
-	FileOwnerUser  = "root"
-	FileOwnerGroup = "root"
-	FilePermission = 0700
+	FileOwnerUserName  = "root"
```

Comment:
I did my best to follow the changes introduced to `azure-operator` here https://github.com/giantswarm/azure-operator/pull/519

I don't know much about file permissions / importance here though, so I mainly just made the operator compile again.